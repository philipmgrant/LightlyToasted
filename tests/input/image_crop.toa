|ImageSource: Z:\images\hero.png
|ImageCrop: circle
testing
with image
cropped to circle