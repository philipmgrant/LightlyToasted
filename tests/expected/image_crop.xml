<toast><visual><binding template="ToastImageAndText04"><image id="1" src="Z:\images\hero.png" hint-crop="circle"/><text id="1">testing</text><text id="2">with image</text><text id="3">cropped to circle</text></binding></visual></toast>